src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const root = document.querySelector('#largeImg');
+
+const onClickHandler = (e) => {
+  e.preventDefault();
+
+  root.src = e.target.parentNode.href;
```

Comment:
What if we will change html structure ?

Let's find specific node using class selector